server/views/pages/goals/vc2GoalsPage.njk

```diff
@@ -35,10 +35,10 @@
       
       <h3 class="govuk-heading-s govuk-!-margin-bottom-1">Employment goals</h3>
       <div class="hmpps-goal-container">
-      {% if (learnerGoals.employmentGoals.length > 0) %}
+      {% if (curiousGoals.employmentGoals.length > 0) %}
```

Comment:
The `learnerGoals` property has been renamed to `curiousGoals` in the view. All the changes from here reflect that rename